docs/rules/prefer-reflect.md

```diff
@@ -54,12 +54,12 @@ Examples of **correct** code for this rule when used without exceptions:
 ```js
 /*eslint prefer-reflect: "error"*/
 
-Reflect.apply(undefined, args);
-Reflect.apply(null, args);
+Reflect.apply(obj.foo, undefined, args);
```

Comment:
IMHO, seems like mdn references `Reflect.apply(Math.floor, undefined, [1.5])` so I personally think `obj.foo` could be used. But i think confirming would be a good idea.

Thanks